### DISCUSSION TITLE
save & read files

### DISCUSSION THREAD

**Discussion by piw-piw** [TurboWarp/packager#328] [Ideas]
Add block which can read and save files from special folder in compiled project.
https://github.com/TurboWarp/packager/issues/110#issuecomment-970473012

**Comment by Squidtito** [comment]
Oh yes. I can already see all the stuff you could do with that

**Comment by Patataman4** [comment]
> Add block which can read and save files from special folder in compiled project. [#110 (comment)](https://github.com/TurboWarp/packager/issues/110#issuecomment-970473012)

I really need this

**Comment by TheShovel** [comment]
This wold have to be Electron only. Plus, you can probably just use js to do that inside the Eval special variable. Just google how to do that with js.

**Reply by Patataman4** [reply]
I didn't understand very well.  do you mean i could use js to read and modify project variables?  and this would make that when starting the game it keeps the data that it had before?

**Reply by TheShovel** [reply]
If you only want only variables, you can use cloud variables and set the host as local. If you want to save a txt file you can modify yes, you can do it with js.

**Reply by Patataman4** [reply]
Now I understand, thank you very much! 😄
